src/app/child-dev-project/notes/model/note.ts

```diff
@@ -158,4 +158,16 @@ export class Note extends Entity {
 
     return false;
   }
+
+  /**
+   * Performs a deep copy of the note copying all simple data
+   * (such as the date, author, e.t.c.) as well as copying the
+   * child-array
+   */
+
```

Comment:
```suggestion
```
Having the comment and the function connected makes it more clear that they belong together